files/porter_bootstrap

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -ex
+#!/bin/bash -e
```

Comment:
@phylake : its printing license key information in /var/log/cloud-init-output.log because we are using -x at the beggining of the init script /usr/bin/porter_bootstrap. For ex: cloudpassage keys are showing up in logs (https://git.corp.adobe.com/apratina/opscode/blob/master/bin/cloudpassage.sh#L14)
![image](https://cloud.githubusercontent.com/assets/4571461/18282707/29ceaf30-7417-11e6-99d7-3285124e8dbd.png)